### PR TITLE
435 [Header] My Account authentication to display logged-in user details

### LINKED ISSUE #435: [Header] My Account authentication to display logged-in user details
To get the first name, last name and email:

- Get the encoded info from the idToken local storage session.
- Decode this ( https://stackoverflow.com/questions/38552003/how-to-decode-jwt-token-in-javascript-without-using-a-library )]
- Use the first name, last name and email in the header.

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #435 

Test URLs:
- Before: https://main--danaher-abcam--aemsites.hlx.page/en-us/technical-resources/guides
- After: https://435-my-account-authentication--danaher-abcam--aemsites.hlx.page/en-us/technical-resources/guides

### CODE REVIEW

**Review by davenichols-DHLS — APPROVED** [review]
LGTM

File: eds/blocks/header/header.js [hunk 1]
```diff
@@ -145,55 +146,140 @@ function countrySelector() {
...
+    return JSON.stringify(tokenId);
+  }
+  return null;
+}
+
+function parsePayload(token) {
+  try {
+    return JSON.parse(atob(token.split('.')[1]));
+  } catch (e) {
+    return null;
```
Comment (rgravitvl) [inline eds/blocks/header/header.js:177]: please do console.error and log a error here

**Review by rgravitvl — APPROVED** [review]
One minor comment, please check. Rest looks good to me.

File: eds/blocks/header/header.js (hunk 1 above)
Comment (sharanagoudapatil) [reply eds/blocks/header/header.js:178]: Done.

**Review by pardeepgera23 — APPROVED** [review]

**Pull request merged**